pkg/background/generate/generate.go

```diff
@@ -351,6 +351,10 @@ func (c *GenerateController) applyGeneratePolicy(log logr.Logger, policyContext
 			ruleNameToProcessingTime[rule.Name] = time.Since(startTime)
 			genResources = append(genResources, genResource)
 		}
+
+		if rule.Generation.GenerateExisting {
+			processExisting = false
+		}
```

Comment:
Same question - do we still need `processExisting`?

Reply:
I think to have a backward compatibility we need this right ?